### ISSUE TITLE
Notes as quick reference guide

### ISSUE THREAD

**Issue by kjk** [kjk/edna#55]
Consider this scenario: you're a JavaScript programmer. You often need to lookup a JavaScript function or topic.

Because we have `Ctrl + B` (`Block: go to` command) a properly formatted note could be a quick reference guide for JavaScript.

Each block would be a reference guide for a single function or topic so let's say you want to lookup `Object.keys()` API:
* you open `JavaScript quick reference` note
* type `Ctrl + B`
* type `keys`, go to block with title `Object.keys` and read the docs

This could be done for every programming language, libraries, html, css, DOM APIs, svelte / vue / react, bun, npm, systemd, tailwindcss etc.

Basically everything that can use a quick reference guide.

If I had a collection of those, I could add a command for people to download the quick ref.

I will prototype this with one of: Go, JavaScript, HTML, CSS
